services/common/src/redux/actionCreators/mineActionCreator.ts

```diff
@@ -92,78 +88,6 @@ export const removeMineType = (mineGuid, mineTypeGuid, tenure) => (dispatch) =>
     .finally(() => dispatch(hideLoading("modal")));
 };
 
-export const createTailingsStorageFacility = (
```

Comment:
Yeah, I remember this bugging me in the past.  Slice is better, and you don't belong here TSF!